src/extension.ts

```diff
@@ -62,6 +68,51 @@ const provideInitialConfigurations = (): string => {
   ].join('\n');
 };
 
+const createTizenProject = async(): Promise<any> => {
+  fs.stat(lastModified.filePath, (err, stat) => {
+    if (lastModified.mtime < stat.mtime) {
+      getPath();
+      setup();
+    }
+  });
+  const createPath = path.join(__dirname, 'CreateTizenProject.sh');
+  const projectName = await vscode.window.showInputBox({
+    placeHolder: 'Tizen IoTjs project name',
+    prompt: 'Name your new project please',
+    ignoreFocusOut: true
+  });
+  if (projectName && tizenStudioPath) {
+    const projectPath = await vscode.window.showOpenDialog({
+      openLabel: 'Set as destination directory',
+      canSelectFiles: false,
+      canSelectFolders: true,
+      canSelectMany: false
+    });
+    if (projectPath) {
```

Comment:
Is it possible to not select a folder with showOpenDialog? If not, the projectPath always has a value and this check is unnecessary.

Reply:
You can cancel the dialog window and in that case projectPath is undefined and you can only create the project if you have a valid projectPath.

Reply:
Could you check the `tizenStudioPath` at the begin of the `createTizenProject` function? You could show a pop-up window (with some information) if the `tizenStudioPath` is not specified.